UI.tsv

```diff
@@ -206,10 +206,10 @@ UI_20150317_000205	Spray Presets
 UI_20150317_000206	{@st49}Select{/}
 UI_20150317_000207	{@st49}Cancel{/}
 UI_20150317_000208	{@st49}Delete{/}
-UI_20150317_000209	Enter the name of the file to save.
+UI_20150317_000209	Enter a file name  to save.
 UI_20150317_000210	{@st43}Friend
 UI_20150317_000211	{@st59}Close Friend List{/}
-UI_20150317_000212	{@st59b} Friend {nl} {@st59s} - Make friends with other players {nl} - Check the status of your friends in List tab {nl} - Check invite, reject, block in Manage tab {/}
+UI_20150317_000212	{@st59b} Friend {nl} {@st59s} - Make friends with other players {nl} - Check the status of your friends in the List tab {nl} - In the Manage tab you can invite, reject or block a player {/}
```

Comment:
https://github.com/Treeofsavior/EnglishTranslation/pulls I've changed Friend to Buddy.